multimedia/yt-dlp/Makefile

```
@@ -0,0 +1,41 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yt-dlp
+PKG_VERSION:=2022.4.8
+PKG_RELEASE:=1
+
+PYPI_NAME:=yt-dlp
+PKG_HASH:=8758d016509d4574b90fbde975aa70adaef71ed5e7a195141588f6d6945205ba
+
+PKG_MAINTAINER:=Michal Vasilek <michal.vasilek@nic.cz>
+PKG_LICENSE:=Unlicense
+PKG_LICENSE_FILES:=LICENSE
+
+include ../../lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/yt-dlp
+  SECTION:=multimedia
+  CATEGORY:=Multimedia
+  TITLE:=Utility to download videos from YouTube.com
+  URL:=https://github.com/yt-dlp/yt-dlp
```

I am thinking, what about adding provides?

```suggestion
  URL:=https://github.com/yt-dlp/yt-dlp
  PROVIDES:=youtube-dl
```